tests/integration_tests/scheduler/test_lsf_driver.py

```diff
@@ -153,3 +157,59 @@ async def finished(iens, returncode, aborted):
 
     await driver.submit(0, f"exit {actual_returncode}")
     await poll(driver, {0}, finished=finished)
+
+
+@pytest.mark.parametrize(
+    "bsub_script, expectation",
+    [
+        pytest.param(
+            "echo 'Job <444> is submitted to default queue'",
+            does_not_raise(),
```

Comment:
Cool trick. I've had to write some tests where I parametrise on whether it throws or not and this is a neat solution.